docs/support/signinissues.md

```diff
@@ -1,12 +1,19 @@
-# Problems with Signing In?
+# Problems with Onboarding and Signing In
 
-## Infinite Sign In Loop
-Your account might be disabled if you encounter an issue where you've signed in but the system keeps routing you back to the page.
+## I received an email that my TechPass account in pending status has been deleted. What should I do?
+When you sign up for TechPass, a TechPass is created for you but will be in pending status until you accept the invitation link we send to you. If you do not accept the invitation link within 30 days, your TechPass account in pending status will be deleted. If you still want to use a TechPass account, sign up again to receive a new invitation link.
+
+## Why is my TechPass account disabled? How to re-enable it?
+Your TechPass account might be disabled if you have not used it for 90 consecutive days. However, when you have not used it for 60 consecutive days, on day 61 you will receive an email alert on your inactive status and also state that your account will be disabled after 30 days. You continue to receive this email until you log in or until your account is disabled.
```

Comment:
The 2nd statement feels a bit lengthy. can consider:
"...90 consecutive days. There will be a email reminder from day 61 on your pending inactive status and remediation actions will be provided. Your account will then be disabled at day 90 should you choose to ignore the reminder emails.

To re-enable or if you think your account ..."

Reply:
done!